File: ReverseRaffle.Data/Raffle.cs
using System;
using System.Collections.Generic;

namespace ReverseRaffle.Data
{
    public class Raffle
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public DateTime EventDate { get; set; }
        public bool IsComplete { get; set; }
        public List<Ticket> Tickets { get; set; }
        public List<Log> Logs { get; set; }
    }
}

File: ReverseRaffle.Web/ClientApp/app/models/app/raffle.ts
import { Ticket } from "./ticket";
import { Log } from "./log";

export class Raffle {
    id: number;
    title: string;
    eventDate: Date;
    isComplete: boolean;
    tickets: Ticket[];
    logs: Log[];

    constructor() {
        this.tickets = new Array<Ticket>();
        this.logs = new Array<Log>();
    }
}